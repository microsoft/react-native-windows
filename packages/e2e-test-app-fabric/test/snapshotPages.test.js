const React = require('react');
import {create, act} from 'react-test-renderer';
import {View} from 'react-native';

test('Component Control', () => {
  const tree = create(<View />).toJSON();
  expect(tree).toMatchSnapshot();
});