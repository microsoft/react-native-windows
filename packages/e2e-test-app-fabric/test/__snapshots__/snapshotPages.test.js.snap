// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Control 1`] = `<View />`;
